repose-aggregator/components/filters/attribute-mapping-policy-validation-filter/src/main/scala/org/openrepose/filters/attributemapping/AttributeMappingPolicyValidationFilter.scala

```diff
@@ -72,11 +69,14 @@ class AttributeMappingPolicyValidationFilter extends Filter with StrictLogging {
             httpServletResponse.sendError(
               SC_UNSUPPORTED_MEDIA_TYPE,
               ucte.message)
-          case e@(_: SaxonApiException | _: TransformerException | _: JsonProcessingException | _: JsonParseException) =>
+          case e@(_: AttributeMapperException) =>
```

Comment:
Why use the muti-exception pattern here now that it is a single?

Reply:
Great question. No reason. Dropping this down to a single exception was one of the last things I did, and it must not have registered that I could simplify this when I dropped the other exception.

Will fix.